### PR TITLE
Publish on JitPack build from kotlin files

### PR DESCRIPTION
**Pull request by yostyle**
Add gradle stuff to build kotlin files and publish on jitpack.

[Jitpack repository](https://jitpack.io/#matrix-org/matrix-analytics-events)
[Jitpack build logs](https://jitpack.io/com/github/matrix-org/matrix-analytics-events/PR81-c0fa6a5b65-1/build.log)

### CODE REVIEW

File: base_types/kotlin/VectorAnalyticsEvent.kt [hunk 1]
```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2021 New Vector Ltd
```
Comment (bmarty) [inline base_types/kotlin/VectorAnalyticsEvent.kt:2]: I believe you have to use the matrix.org copyright in this project (applicable to all added files)

**Review by bmarty — COMMENTED** [review]
Adding the android project at the root of this project is a bit dirty, you should maybe create a subfolder `android` .

File: base_types/kotlin/VectorAnalyticsEvent.kt (hunk 1 above)
Comment (yostyle) [reply base_types/kotlin/VectorAnalyticsEvent.kt:2]: I applied the same header as the classes in [types/kotlin2](https://github.com/matrix-org/matrix-analytics-events/tree/jitpack/types/kotlin2).
I could apply this change but I would have to do the same thing for all generated classes.
WDYT

**Comment by yostyle** [comment]
> Adding the android project at the root of this project is a bit dirty, you should maybe create a subfolder `android` .

@bmarty build.gradle.kts is not detected by jitpack in a subdirectory. I had to migrate to build.gradle with groovy language.

**Review by bmarty — APPROVED** [review]
Thanks, LGTM. We may iterate on package name and copyright later.

**Pull request merged**